app/views/orders/order_confirmation.html.erb

```diff
@@ -39,10 +39,19 @@
     </tr>
 
     <tr>
+      <td>Shipping Method</td>
+      <td><%= @order.shipping_method %></td>
+    </tr>
+    <tr>
+      <td>Shipping Cost</td>
+      <td><%= to_money(@order.shipping_cost/100.0) %></td>
```

Comment:
On this page (`/order/16/updateshipping`), it looks like `@order.shipping_cost` is already in dollars. My $90 shipping is listed as $0.90. The total listed at the bottom is the accurate amount, just not what is listed as the Shipping Cost.